thirdparty/patches/grpc-cython-copts.patch

```diff
@@ -30,6 +30,8 @@ diff --git bazel/cython_library.bzl bazel/cython_library.bzl
 +            srcs = [stem + ".cpp"] + cc_kwargs.pop("srcs", []),
```

Comment:
can you tell me why we need these changes?

Reply:
Existing patches, I don't know why we need it in the first place.